packages/sdks/snippets/qwik-city/src/components/NavBar.tsx

```diff
@@ -0,0 +1,18 @@
+import { component$ } from '@builder.io/qwik';
+import type { BuilderContent } from '@builder.io/sdk-qwik';
+
+export const NavBar = component$((props: { links: BuilderContent | null }) => {
```

Comment:
you're typing links as `BuilderContent | null` but it will only get passed to this component when links is defined because of `{links.value && <NavBar links={links.value} />}`